k8s/eks/testground_install.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+set -e # Let's make sure that exit code 1 drops the script
```

Comment:
Could you add other, good practice flags?

I expect at least (see my other comments, we are using undefined variables)

```bash
set -u
set -o pipefail
```

Reply:
Added to the script.